Update README to point to r2u (#86)
Enchufa2/bspm: README.md

### Ubuntu

> There are thousands of binary packages available via the

Reviewer: Can we improve that line?  
```suggestion
(Essentially) all of CRAN is available as binary packages via the
```

[r2u](https://eddelbuettel.github.io/r2u/) repo:

```bash
$ . /etc/os-release # to get UBUNTU_CODENAME
$ URL="https://raw.githubusercontent.com/eddelbuettel/r2u/master/inst/scripts"

> $ curl -s "${URL}/add_cranapt_${UBUNTU_CODENAME}.sh" | sudo bash -s

Reviewer: Nifty three lines.  I like it.
